provider/provider_type/provider_type.go

```diff
@@ -31,6 +31,8 @@ const (
 	HDFS             Type = "HDFS"
 	AZURE            Type = "AZURE"
 	UNIT_TEST        Type = "UNIT_TEST"
+
```

Comment:
curious what is the reasoning behind the extra space?

Reply:
just to separate it all from the rest